src/test/java/org/takes/http/BkParallelTest.java

```diff
@@ -80,12 +83,16 @@ public Response act(final Request req)
             }
         };
         new Thread(
+            // @checkstyle AnonInnerLengthCheck (23 lines)
             new Runnable() {
                 @Override
                 public void run() {
                     try {
                         new FtBasic(
-                            new BkParallel(new BkBasic(take)),
+                            new BkParallel(
+                                new BkBasic(take),
+                                count
```

Comment:
@lautarobock but you will always have 1 or 2 or 3 threads, not more. now we always have 3. how do 1 or 2 solve the problem?